content/pages/meetup/index.rst

```diff
@@ -6,8 +6,14 @@ meetup
 :url: meetup
 :save_as: meetup/index.html
 
-meetup 2023
+Meetup 2023
 ==============
 
 * `Meetup 2023 March at NYCU
-  <{filename}2023/03-nycu.rst>`__
\ No newline at end of file
+  <{filename}2023/03-nycu.rst>`__
+
+Meetup 2022
+==============
+
+* `Sciwork Meetup 2022 October in NYCU
```

Comment:
There should not be the old content.

Reply:
Yes, I did merge it from `develop`. Would it be acceptable to delete the lines from `index.rst` with just a simple deletion?

Reply:
okay.